.github/workflows/license_check.yml

```diff
@@ -0,0 +1,25 @@
+name: Scancode
+
+on: [pull_request]
```

Comment:
I added `[pull_request, push]`. I know we don't ever have push's but figured its better to have it in here.

Reply:
this action only acts on changed files, on push we will need to scan the whole tree which is fine I guess, but I will need to update a few things to make this work.

Reply:
Ah, that makes sense.